modules/installation-configuration-parameters.adoc

```diff
@@ -182,16 +182,11 @@ endif::ibm-power-vs[]
 
 You can customize your installation configuration based on the requirements of your existing network infrastructure. For example, you can expand the IP address block for the cluster network or provide different IP address blocks than the defaults.
 
-// OSDOCS-1640 - IPv4/IPv6 dual-stack bare metal only
-// But only for installer-provisioned
-// https://bugzilla.redhat.com/show_bug.cgi?id=2020416
-// Once BM UPI supports dual-stack, uncomment all the following conditionals and blocks
-
-ifndef::agent,bare,ibm-power,ibm-z,vsphere[]
+ifndef::agent,bare,ibm-power,ibm-z,vsphere,osp[]
```

Comment:
The ifndef block means that the `Only IPv4 addresses are supported.` statement won't be included in the OpenStack docs.